packages/genesys-spark-components/src/components/legacy/gux-action-toast-legacy/example.html

```diff
@@ -1,4 +1,25 @@
-<h1>gux-action-toast</h1>
+<h1>gux-action-toast-legacy</h1>
+
+<div>
+  <gux-icon
+    style="color: #6c5619"
+    icon-name="alert-warning-triangle"
+    decorative="true"
+  ></gux-icon>
+  <span
+    >Avoid using legacy components. Legacy components may not be supported in
+    future major releases. We recommend using the
+  </span>
+  <a href="/#gux-toast" target="_blank">gux-toast</a>
+  <span>component instead. See the </span>
+  <a
+    href="https://github.com/MyPureCloud/genesys-spark/tree/main/packages/genesys-spark-components/documentation/migrations/v4"
```

Comment:
Should we reference the gux-toast section of the migration v4 guide instead of referencing the root level here?

Reply:
I didn't want to link to something that we reorganize in the future. We can circle back and make these links more specific once the structure of the migration guide files are solidified